cmd/podman/cp.go

```diff
@@ -84,10 +83,14 @@ func copyBetweenHostAndContainer(runtime *libpod.Runtime, src string, dest strin
 	if len(srcPath) == 0 || len(destPath) == 0 {
 		return errors.Errorf("invalid arguments %s, %s you must specify paths", src, dest)
 	}
+
 	ctr := srcCtr
 	isFromHostToCtr := ctr == nil
 	if isFromHostToCtr {
 		ctr = destCtr
+		destPath = anchorContainerPath(destPath)
+	} else {
+		srcPath = anchorContainerPath(srcPath)
```

Comment:
Doesn’t this completely break the use of in-container relative paths (the code involving `ctr.WorkingDir` below)?

Reply:
@mtrmac I was following the contract with docker cli where cp is documented as anchored at the container's root.

Reply:
OK, behaving the same makes sense; still, the code should be simplified.
- I don’t see any reason to do the “anchor” step up here when it could just as well be done in the following `if isFromHostToCtr` statement that deals with all other path remapping, and consumes the value.
- The `if isFromHostToCtr` case still has an `if filepath.IsAbs(destPath)` / `else`, with the two behaving differently (and the second one probably dead)
- The `if !isFromHostToCtr` case has an `if filepath.IsAbs(srcPath)` check with no `else`. Sure, that technically can’t happen right now, but it’s confusing and risky to leave around that way.